### PR TITLE
Remove unnecessary logging

### PR DESCRIPTION
**Pull request by johanflint**
Use breakpoints in your browser instead.

### CODE REVIEW

File: src/WidgetName/widget/WidgetName.js [hunk 1]
```diff
@@ -126,7 +123,7 @@ define([
                             //TODO what to do when all is ok!
                         },
                         error: dojoLang.hitch(this, function(error) {
-                            console.log(this.id + ": An error occurred while executing microflow: " + error.description);
+                            console.error(this.id + ": An error occurred while executing microflow: " + error.description);
```
Comment (dson) [inline src/WidgetName/widget/WidgetName.js]: Console messages are not visible to the user. I think it's better to either remove the error handler completely (the client will then take care of it) or use mx.ui.error to show a message to the user.

**Comment by Andries-Smit** [comment]
Or use the  "mendix/logger"
that can output the messages in debug mode

**Comment by Andries-Smit** [comment]
@JelteMX The API does not allow to use the logger the AMD way
The only way is to use the global definded 
```javascript
logger.log("Hello World!")
```
Would it better to allow it in the api and get the logger in the AMD way?
```javascript
require("mendix/logger", function(logger){
    logger.log("Hello World!")
 })
```

**Comment by JelteMX** [comment]
@Andries-Smit I don't know if it's possible get the logger in the AMD way. I'll ask around.

The logger is always exposed globally. ```logger.log``` is not available by the way. For the logger we defined a couple of loglevels:

```var ALL = 0, DEBUG = 1, INFO = 2, WARN = 3, ERROR = 4;```

The logger methods are basically wrappers for ```console```. In the AppStoreWidgetBoilerplate (and what we now do consistently) is use ```logger.debug``` and setting ```logger.level(logger.DEBUG)``` if needed. This way a client's console isn't flooded with log messages if it isn't needed

**Pull request closed without merging**